FIAT/hu_zhang.py

```diff
@@ -54,23 +55,30 @@ def __init__(self, ref_el, degree, variant, qdegree):
             entity_ids[1][entity].extend(range(cur, len(nodes)))
 
         # interior dofs
-        cur = len(nodes)
```

Comment:
is the point variant going away?

Reply:
No, this block is only dealing with setting up a quadrature for the integral case